app/controllers/application_controller.rb

```diff
@@ -1,4 +1,4 @@
-class ApplicationController 
+class ApplicationController < ActionController::Base
   # Only allow modern browsers supporting webp images, web push, badges, import maps, CSS nesting, and CSS :has.
-  allow_browser versions: :modern
+  # allow_browser versions: :modern
```

Comment:
Why did you remove this line?
```suggestion
  allow_browser versions: :modern
```